src/file_name_plugin/__tests__/plugin.test.ts

```diff
@@ -1,7 +1,33 @@
 import { KEYS } from 'jest-watcher';
 import type { Config } from '@jest/types';
-import pluginTester from '../../test_utils/pluginTester';
-import FileNamePlugin from '../plugin';
+import { jest } from '@jest/globals';
+
+let pluginTester: typeof import('../../test_utils/pluginTester').default = null;
+let FileNamePlugin: typeof import('../plugin').default = null;
+
+jest.unstable_mockModule('ansi-escapes', () => ({
+  default: {
+    clearScreen: '[MOCK - clearScreen]',
+    cursorDown: (count = 1) => `[MOCK - cursorDown(${count})]`,
+    cursorLeft: '[MOCK - cursorLeft]',
+    cursorHide: '[MOCK - cursorHide]',
+    cursorRestorePosition: '[MOCK - cursorRestorePosition]',
+    cursorSavePosition: '[MOCK - cursorSavePosition]',
+    cursorShow: '[MOCK - cursorShow]',
+    cursorTo: (x, y) => `[MOCK - cursorTo(${x}, ${y})]`,
+  },
+}));
+
+jest.doMock('ansi-escapes', () => ({
```

Comment:
needs to mock the CJS version used by `jest-watcher` _and_ the ESM version used by ourselves